roles/edpm_libvirt/templates/libvirt_virtlogd/libvirt_virtlogd.json.j2

```diff
@@ -8,6 +8,11 @@
         "KOLLA_CONFIG_STRATEGY":"COPY_ALWAYS"
     },
     "volumes": [
+{% if edpm_libvirt_tls_certs_enabled %}
```

Comment:
i need to look back at the nova changes i guess but one thing we are trying to avoid if possible is making this list dynamic


we would generally prefer to always passthrough the folder even if its empty

so i would generally prefer if this was just 

/var/lib/openstack/certs/libvirt/:/etc/pki/libvirt/:ro

and if we set the filet to have the correct name instead when we copy it

were tlse is not enabled it would just end up being an empty dir.